engine/src/main/java/org/terasology/input/events/InputEvent.java

```diff
@@ -50,6 +58,10 @@ public Vector3i getTargetBlockPosition() {
         return targetBlockPosition;
     }
 
+    /**
+     *
+     * @return The time since the event was fired (also the game update loop's delta time).
```

Comment:
What is the connection between the update delta and this one? Is the event's delta always updated, or is it just the delta of the update pass processing it?